server/bucket_notifications/README.md

```diff
@@ -0,0 +1,8 @@
+# Bucket Notifications Plugin
+
+A standalone girder plugin for reacting to GCS Bucket PubSub notifications.
+
+## Setting up a bucket
+
+* See the [Google Cloud Docs](docs/Google-Cloud.md)
+* [Check the official Google Cloud Docs](https://cloud.google.com/storage/docs/pubsub-notifications)
```

Comment:
I'm going to set up a test instance that we can use to verify the correctness of this new plugin.